plugin-testing-framework/script/test.ts

```diff
@@ -0,0 +1,470 @@
+"use strict";
```

Comment:
I feel like this file could be broken down better. The server related code can go in one file, the test runner in one file, and the test API (describe, it) can go into another file.